docs/DEVELOPING.md

```diff
@@ -441,6 +441,106 @@ $ make ftest NAME=sharepoint
 
+## ServiceNow Connector
+
+The [Elastic ServiceNow connector](../connectors/sources/servicenow.py) is provided in the Elastic connectors python framework and can be used as a [connector client](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+All services and records the user has access to will be indexed according to the configurations provided.
+
+### Usage
+
+To use this connector as a **connector client**, use the **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+### Compatibility
+
+ServiceNow "Tokyo", "San Diego" & "Rome" versions are compatible with Elastic connector frameworks.
+
+### Configuration
+
+The following configuration fields are required to set up the connector:
+
+##### `url`
+
+The host url of the ServiceNow instance.
+
+##### `username`
+
+The username of the account for ServiceNow.
+
+##### `password`
+
+The password of the account used for ServiceNow.
+
+##### `services`
+
+Comma-separated list of services to fetch data from ServiceNow. If the value is `*`, the connector will fetch data from the list of basic services provided by ServiceNow: 
+- [User](https://docs.servicenow.com/bundle/utah-platform-administration/page/administer/roles/concept/user.html)
+- [Incident](https://docs.servicenow.com/bundle/tokyo-it-service-management/page/product/incident-management/concept/c_IncidentManagement.html)
+- [Requested Item](https://docs.servicenow.com/bundle/tokyo-servicenow-platform/page/use/service-catalog-requests/task/t_AddNewRequestItems.html)
+- [Knowledge](https://docs.servicenow.com/bundle/tokyo-customer-service-management/page/product/customer-service-management/task/t_SearchTheKnowledgeBase.html)
+- [Change Request](https://docs.servicenow.com/bundle/tokyo-it-service-management/page/product/change-management/task/t_CreateAChange.html)
+
+Default value is `*`. Examples:
+
+  - `User, Incident, Requested Item, Knowledge, Change Request`
+  - `*`
+
+##### `retry_count`
```

Comment:
`retry count` and `concurrent downloads` should be advanced config, correct?

Reply:
Yes, correct they are part of advance config. User will not be able to see these fields in UI.